package-lock.json

```diff
@@ -1284,7 +1284,6 @@
       "requires": {
         "anymatch": "https://registry.npmjs.org/anymatch/-/anymatch-1.3.0.tgz",
         "async-each": "https://registry.npmjs.org/async-each/-/async-each-1.0.1.tgz",
-        "fsevents": "https://registry.npmjs.org/fsevents/-/fsevents-1.0.15.tgz",
```

Comment:
Are you by any chance using Windows or Linux? If so, please update npm to [5.6.0](http://blog.npmjs.org/post/167963735925/v560-2017-11-27) and regenerate `package-lock` without removing fsevents.

Reply:
Using debian indeed.